spec/requests/v0/user_spec.rb

```diff
@@ -11,10 +11,12 @@
     let(:mhv_user) { build(:user, :mhv) }
     let(:v0_user_request_headers) { {} }
     let(:edipi) { '1005127153' }
+    let(:mhv_user_verification) { create(:mhv_user_verification) }
 
     before do
       allow(SM::Client).to receive(:new).and_return(authenticated_client)
       allow_any_instance_of(MHVAccountTypeService).to receive(:mhv_account_type).and_return('Premium')
+      allow_any_instance_of(User).to receive(:user_verification).and_return(mhv_user_verification)
```

Comment:
I'd like to be able to get this spec working without stubbing this. Is the issue that the `user.mhv_correlation_id` is blank for the breaking spec?

As a backup, if you do:

`let(:mhv_user_verification) { create(:mhv_user_verification, backing_idme_uuid: mhv_user.idme_uuid) }`

Then that should properly create a user_verification associated to the user.

Also, you might need to add a `!` so it automatically runs the code since you might not refer to it, so:
`let!(:mhv_user_verification) ...`

Reply:
Yeah I have fought over this one for a LONG time.  This suggested change causes 12 test failures

Reply:
oh I'll pull the branch and try

Reply:
Gave a suggestion to fix most the specs, and then to fix the remaining spec, put this inside the `with missing MHV accounts` context:
`let!(:mhv_user_verification) { create(:mhv_user_verification, backing_idme_uuid: mhv_user.idme_uuid) }`

Reply:
that'll fix all the specs. 

Makes sense cause most specs have an `mhv_correlation_id`, so the `UserVerification` is matched on that. And then that single spec is a messed up MHV user with the `mhv_correlation_id` cleared out, so that uses the backup mechanism to associate the `UserVerification` which is the `backing_idme_uuid`

Reply:
Did this resolve the failures for you, Trevor?  I still have 12 failures.  I just pushed my code up